ArticleTemplates/assets/js/bootstraps/football.js

```diff
@@ -39,28 +39,28 @@ function footballChart(homeTeam, awayTeam) {
     // Background
     const backgroundData = [['null', 100]];
 
-    const backgroundarc = svg.arc()
+    const backgroundarc = arc()
```

Comment:
Same question about whether this should be `arcShape`?

Reply:
I don't think so because `arcShape` and `backgroundarc` are two different functions:

```
const arcShape = arc()
        .outerRadius(radius)
        .innerRadius(radius / 3);

 const backgroundarc = arc()
        .outerRadius(radius - 1)
        .innerRadius((radius / 3) + 1);
```